stacks/vcluster/overlays/staging/kustomization.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
I'd like to give external-secret one more chance before going with that approach.